feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/components/TimetableItemDetailFloatingMenu.kt

```diff
@@ -98,14 +100,14 @@ private fun TimetableItemDetailFloatingActionButtonMenu(
     }
 
     val roomTheme = LocalRoomTheme.current
-    val menuItemContainerColor = roomTheme.primaryColor // TODO: use room containerColor
+    val menuItemContainerColor = roomTheme.primaryColor.copy(alpha = 0.5f).compositeOver(Color.Black)
```

Comment:
If the button background is transparent, the text will be difficult to read, so I have placed a black background.